src/services/flatmoney/stableLp.ts

```diff
@@ -0,0 +1,36 @@
+import { ethers } from "../..";
+import DelayedOrderAbi from "../../abi/flatmoney/DelayedOrder.json";
+
+export function getAnnounceStableDepositTxData(
+  depositAmount: ethers.BigNumber | string,
+  minAmountOut: ethers.BigNumber | string,
+  keeperFee: ethers.BigNumber | string
```

Comment:
Do you think we could calculate the minAmountOut internally by just providing a slippage percentage?
Maybe keeperFee could also be derived?

Reply:
yes, keeperFee and slippage are derived now if you look at the pool functions. 9a99688